### DISCUSSION TITLE
Account flagged | Bug

### DISCUSSION THREAD

**Discussion by kadirwinchester** [community/community#138165] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Hello, I've noticed that my GitHub account has been mistakenly flagged as spam.
I believe this is an error. I kindly request that my account be reopened.
Could you please assist me with this issue? Thank you in advance for your support.
Best regard.
GitHub Username Abdulkadirbulbul

**Comment by rioyuda02** [comment]
Hi bro,
I think you must create tickets to GitHub Support Team.
Link : https://support.github.com/tickets/personal/0

**Comment by kadirwinchester** [comment]
Hello, I opened a ticket from my own spam account. Even though my account
is Github Pro, I have not received any response for 1.5 months. The ticket
is still open. This really disappointed me.
There was no reason for me to get spammed, because I followed all the
rules.

The username of my account that got spammed is @Abdulkadirbulbul


[image: image.png]


Rio Yudayanto ***@***.***>, 9 Eyl 2024 Pzt, 11:04 tarihinde
şunu yazdı:

> Hi bro,
> I think you must create tickets to GitHub Support Team.
> Link : https://support.github.com/tickets/personal/0
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/orgs/community/discussions/138165#discussioncomment-10587890>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/BKSPHJI3VVW35WXMAFZ2EC3ZVVJBBAVCNFSM6AAAAABN35P7FOVHI2DSMVQWIX3LMV43URDJONRXK43TNFXW4Q3PNVWWK3TUHMYTANJYG44DSMA>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>


--

**Reply by ebndev** [reply]
Hi @kadirwinchester,

This community is managed by GitHub employees, but it is not a support forum. You have done everything necessary on your end by opening a ticket on our [Support page](https://support.github.com/contact).

This is a public forum, and the community is unable to assist with nor escalate your ticket, but we can assure you it is in the right place. It will be answered in the order it was received based on our team's resources and the amount of tickets ahead of yours.

As no other Community Discussions team members will be able to provide additional help with account related questions I am going to close this discussion. We appreciate your understanding!
